container/docker/factory.go

```diff
@@ -340,7 +340,10 @@ func Register(factory info.MachineInfoFactory, fsInfo fs.FsInfo, ignoreMetrics c
 			glog.Errorf("devicemapper filesystem stats will not be reported: %v", err)
 		}
 
-		status := StatusFromDockerInfo(*dockerInfo)
+		status, err := StatusFromDockerInfo(*dockerInfo)
```

Comment:
This actually seems concerning, as we only call `Register` once.  If it times out the first time, then you get no disk stats on DM.

Reply:
Although I suppose it was still possible for it to fail before.  Now it is just more likely since we have added a timeout.

Reply:
nvm.  Can you change this to ignore the error?  It should never fail to populate `status.DriverStatus` if you reorder the operations above as I requested.